mlir/test/lib/Dialect/Tosa/TosaTestPasses.cpp

```diff
@@ -165,13 +165,18 @@ ConvertTosaConv2DOp::matchAndRewrite(Operation *op,
   // Obtain the quantized scale = multiplier and shift.
   computeMultiplierAndShift(opTensorScale, multiplier, shift, 32);
 
+  bool input_unsigned =
+      newTosaConv2DOp.getResult().getType().isUnsignedInteger();
+  bool output_unsigned = outputType.isUnsignedInteger();
+
```

Comment:
Nit: does `outputType == newTosaConv2DOp.getResult().getType()` here?

Reply:
Was just thinking if that's the case, we can simplify the above code

Reply:
Hi @psunn do you still plan to simplify the code here?

Reply:
Will simplify this nit in future PRs.